routes/api.php

```diff
@@ -24,6 +24,63 @@
 
 
 // api to fetch all todo tasks
+Route::get('task', [TaskController::class, 'index']);
```

Comment:
Please most of the route here are duplicates
pull from the upstream and then accept incoming changes